etl/steps/data/garden/state_capacity/2023-10-19/state_capacity_dataset.meta.yml

```diff
@@ -0,0 +1,447 @@
+
+
+          The data cover 72 countries from 2005-2015.
+        processing_level: minor
+        display:
+          name: Quality of Budgetary and Financial Management
+          numDecimalPlaces: 1
+      irai_qpa:
+        title: Quality of Public Administration (CPIA)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the extent to which civilian central government staff are structured to design and implement government policy and deliver services effectively. It ranges from 0 to 4 (greater quality of public administration).
+        description_from_producer: >-
+          In this scale, recoded to run from 0 to 4, World Bank staff rate countries on the extent to which
+          civilian central government staff are “structured to design and implement government policy and
+          deliver services effectively.” The criteria involve:
+          - the extent to which there are effective coordination mechanisms that ensure policy consistency across departmental boundaries.
+          - the extent to which organizational structures follow functional lines and avoid duplication
+          - the extent to which hiring and promotion are based on merit and performance
+          - the extent to which pay and benefit levels to not discourage people from working in the
+          public sector.
+          - the sustainability of the public wage bill
+
+
+          The data cover 72 countries from 2005-2015.
+        processing_level: minor
+        display:
+          name: Quality of Public Administration
+          numDecimalPlaces: 1
+      irai_erm:
+        title: Efficiency of Revenue Mobilization (CPIA)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the extent to which the tax system and tax administration are efficient. It ranges from 0 to 4 (greater efficiency of revenue mobilization).
+        description_from_producer: >-
+          This is a five-point scale (rescaled 0 to 4) in which World Bank staff rate countries on tax systems
+          and tax administration. Higher scores are given to higher-quality systems of taxation. Criteria
+          include:
+          - the extent to which the tax laws are “stable, certain, and clear.”
+          - the extent to which tax administration is “effective, and entirely rule-based.”
+          - administrative and compliance costs are low, and information systems are functioning.
+          - the extent to which taxes come from sales/VAT taxes, property taxes, and corporate/personal
+          income taxes applied to a broad base, rather than from taxes on international trade.
+
+
+          For further details, see World Bank (2017) p. 39. Coverage is 2005-2015.
+        processing_level: minor
+        display:
+          name: Efficiency of Revenue Mobilization
+          numDecimalPlaces: 1
+      milexpercap:
+        title: (log) Military expenditures per capita (SIPRI/COW)
```

Comment:
I don't like this "(log)" in the title. Why don't you instead use `presentation.title_variant: "In logarithmic units"` (or something like that). Similarly, instead of mentioning the producer in each title, you could use `presentation.attribution_short: "SIPRI/COW"`. However, I think you can't have multiple indicators with the same title, unfortunately. But you could give the indicator a nicer name, like "Military expenditures per capita in logarithmic scale, according to SIPRI/COW".
The same applies to other indicators. I leave this up to you.